pkgs/development/mobile/androidenv/androidndk-pkgs.nix

```diff
@@ -91,8 +91,7 @@ rec {
           $out/bin/${stdenv.targetPlatform.config}-c++ \
           $out/bin/${stdenv.targetPlatform.config}-gcc \
           $out/bin/${stdenv.targetPlatform.config}-g++ \
-          -e '130i    extraBefore+=(-Wl,--fix-cortex-a8)' \
-          -e 's|^(extraBefore=)\(\)$|\1(${builtins.toString flags})|'
+          -e 's|^(extraBefore=)\((.*)\)$|\1(\2 -Wl,--fix-cortex-a8 ${builtins.toString flags})|'
```

Comment:
This doesn't make the situation any worse, so it should be fine. On the other hand, I don't understand why we don't use $out/nix-support/cc-cflags here. Even better just move these flags to cc-wrapper /cc @Ericson2314 

Related to https://github.com/NixOS/nixpkgs/pull/56649

Reply:
@matthewbauer yeah tbh I forgot about `$out/nix-support/cc-cflags`. Either is fine, but the more ambitious stuff should be done on master not 18.09